ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -92,6 +96,14 @@ public final class SystemColumns {
           true,
           true,
           true
+      ),
+      PseudoColumn.of(
+          ROWID_NAME,
+          ROWID_TYPE,
+          ROWID_PSEUDOCOLUMN_VERSION,
+          true,
+          true,
+          true
```

Comment:
Why is ROWID disallowed from pull queries? I don't see a technical reason to disallow this, since we have access to the key bytes in state stores. Are we saying that users won't be interested in this?

Reply:
Also, if it's indeed expected that ROWID cannot be used in pull queries, then we should have an RQTT to check that the error message is reasonable.

Reply:
Yeah that is a valid point, I think I had just followed what `ROWOFFSET/ROWPARTITION` did, but I can try allowing it in pull queries and make sure it works as expected